minix.sh

```diff
@@ -298,6 +299,10 @@ do
 				break
 			done
 			
+			;;
+		"7")
+			sudo runuser -u $SUDO_USER -- code -w $MINIX_USR_LOCAL_DIR
```

Comment:
Hmm, może to moment żeby scalić opcję 7 i g w jedną, bazującą na jakimś $EDITOR?

Reply:
Dodałem zmienną $EDITOR, domyślnie gedit
Do zaopiniowania: 
- wartość $EDITOR pobrana z shella 
- wsparcie dla innych IDE niż vscode